### PR TITLE
Base account screens

### PR DESCRIPTION
**Pull request by jetpack3331**
Added:
- Login screen
- Prompt for encryption password
- Mocked eth instance and call/listen actions
- Loading screen
- some updates...

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-version: '3.3'
+version: '2.0'
```
Comment (florianlenz) [inline docker-compose.yml]: version 3.3 is on purpose since it support's caching of volumes which speeds up the development on mac/windows.

Comment (florianlenz) [reply docker-compose.yml]: What is the benefit of setting it to 2.0?

File: preview/index.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+export default class Preview extends React.Component {
+
+  render() {
+    return (
+      <div>
+        abcd
```
Comment (florianlenz) [inline preview/index.js:9]: Is this test data? :D

Comment (jetpack3331) [reply preview/index.js:9]: Nope :) It's prepared for preview page of components

File: docker-compose.yml (hunk 1 above)
Comment (jetpack3331) [reply docker-compose.yml]: When i installed on fresh system docker libraries, it wasn't supporting version 3
And i left it accidentaly, will get back that

**Pull request closed without merging**